modules/transactions/src/testFixtures/java/org/apache/ignite/internal/tx/storage/state/test/TestConcurrentHashMapTxStateStorage.java

```diff
@@ -57,13 +57,17 @@ public boolean compareAndSet(UUID txId, TxState txStateExpected, @NotNull TxMeta
             if (old == null && txStateExpected == null) {
                 TxMeta oldMeta = storage.putIfAbsent(txId, txMeta);
                 if (oldMeta == null) {
```

Comment:
For example, this check is not needed, let's do something like `TxStateRocksDbStorage#compareAndSet`

Reply:
Fix it